lib/gettext_i18n_rails/backend.rb

```diff
@@ -23,7 +23,7 @@ def translate(locale, key, options)
           interpolate(translation, options)
         else
           result = backend.translate(locale, key, options)
-          (RUBY19 and result.is_a?(String)) ? result.force_encoding("UTF-8") : result
+          (RUBY19 and result.is_a?(String)) ? result.dup.force_encoding("UTF-8") : result
```

Comment:
I considered adding a conditional to `dup` only if `frozen?` but wasn't sure if we were intentionally modifying the object in place or not, perhaps for memory reasons.  Let me know if there is a preference.